tests/main/interfaces-cups/task.yaml

```diff
@@ -0,0 +1,68 @@
+summary: Ensure that the cups interfaces work with app providers
+
+systems: [ubuntu-*]
+
+details: |
+    A snap providing the cups-control and cups interfaces should be able to
+    create the control socket, with connecting consuming snaps able to use it.
+    This intentionally does not test the mediation properties of the cupsd
+    server.
+
+prepare: |
+    #shellcheck source=tests/lib/snaps.sh
+    . "$TESTSLIB"/snaps.sh
+    install_local test-snapd-provider
+    install_local test-snapd-consumer
+
+    if [ -e /run/cups ]; then
+        mv /run/cups /run/cups.orig
```

Comment:
This should use the new fs-state helper. Probably only in master where the helper is landing.